tests/utils.py

```diff
@@ -75,3 +77,10 @@ def mock_reverse(view_name, args=None, kwargs=None, request=None, format=None):
 
 def fail_reverse(view_name, args=None, kwargs=None, request=None, format=None):
     raise NoReverseMatch()
+
+
+def remove_object_addresses(repr_str):
+    """
+    Remove object address hex strings from reprs, for stable comparison.
+    """
+    return re.sub(" at 0x[0-9A-F]+>", ">", repr_str)
```

Comment:
I'll have to dig in a little deeper to this stuff, as `rest_framework.utils.representation.smart_repr()` should have already done the hex removal (heh), but for some reason always doesn't.